ckcvisualizer/org.js

```diff
@@ -162,7 +246,7 @@ function drawOrgChart(dataInput){
       function drawChart() {
           
         //create the list of colors that corresponde to each node
-        var colorList = makeColorsList(dataInput);
```

Comment:
Should probably include a comment here to specify that this is where ic can be changed beck to spectrum colorList with a single line...